ibm_mq/datadog_checks/ibm_mq/config.py

```diff
@@ -85,7 +85,9 @@ def __init__(self, instance):
         self.collect_statistics_metrics = is_affirmative(
             instance.get('collect_statistics_metrics', False)
         )  # type: bool
-
+        self.collect_reset_queue_metrics = is_affirmative(
+            instance.get('collect_reset_queue_metrics', True)
```

Comment:
Note to reviewer: I set `collect_reset_queue_metrics` to default to `True` to retain backwards compatibility. However, this also means a newly installed IBM MQ integration with read-only permissions will report the `MQRC_NOT_AUTHORIZED` warning unless `collect_reset_queue_metrics` is set to `False` manually. Should this option default to `False` instead?

Reply:
I think it's ok to leave it enabled by default